concordium-consensus/tests/consensus/ConcordiumTests/KonsensusV1/Consensus/Blocks.hs

```diff
@@ -0,0 +1,1212 @@
+{-# LANGUAGE DataKinds #-}
+{-# LANGUAGE NumericUnderscores #-}
+
+module ConcordiumTests.KonsensusV1.Consensus.Blocks (tests) where
```

Comment:
Would it be possible to describe what the intent of this test is? The name is rather generic, and there are a lot of different test cases, so it is a bit unclear from those what the intended scope is.